src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,14 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop): Use abstract reference instead of specific one where possible:

Reply:
Abstract reference is used instead of concrete